src/util.ts

```diff
@@ -524,6 +524,43 @@ export function expandMultilink(link: string): string {
 	return url.href
 }
 
+export function combineRaffleLink(links: string[]): string {
```

Comment:
I'd add a 1 sentence jsdoc-type comment on function description, because at first glance idk what this does